docs/docs/reference/crd-reference/taskdefinition.md

```diff
@@ -270,7 +270,7 @@ spec:
       possibly with different parameters
       that are set in the calling `KeptnTaskDefinition` resource.
 
-      You must annotate the `KeptnApp` resource to run the
+      You must create the `KeptnAppContext` resource to run the
```

Comment:
does this sentence even make sense here? Not sure I get what that means in this context

Reply:
hmm good catch, I will look at this part

Reply:
adapted